locales/es.po

```diff
@@ -0,0 +1,252 @@
+"Dolphin y archivos transmitidos desde Slippi.gg."
+
+#: src/renderer/app/create.messages.ts:3:18
+#: src/renderer/pages/settings/create.messages.ts:4:18
+msgid "Replays"
+msgstr "Repeticiones"
+
+#: src/renderer/components/path_input/path_input.messages.ts:2:17
+msgid "Select"
+msgstr "Seleccionar"
+
+#: src/renderer/pages/settings/advanced_app_settings/language_selector/language_selector.messages.ts:3:33
+msgid "Select your preferred language for the app."
+msgstr "Selecciona tu idioma preferido para la aplicación."
+
+#: src/renderer/app/header/header.messages.ts:7:19
+msgid "Settings"
+msgstr "Configuración"
+
+#: src/renderer/pages/home/news_feed/news_feed.messages.ts:6:19
+msgid "Show more"
+msgstr "Mostrar más"
+
+#: src/renderer/app/create.messages.ts:4:19
+msgid "Spectate"
+msgstr "Espectador"
+
+#: src/renderer/pages/settings/advanced_app_settings/advanced_app_settings.messages.ts:8:28
+msgid "Successfully cleared temporary files."
+msgstr "Archivos temporales eliminados correctamente."
+
+#: src/renderer/pages/settings/game_settings/game_settings.messages.ts:3:34
+msgid "The path to an NTSC Melee 1.02 ISO."
+msgstr "La ruta a un archivo ISO de Melee NTSC 1.02."
+
+#: src/renderer/pages/replays/replay_browser/replay_browser.messages.ts:5:41
+#. {0} readableBytes: string
+msgid "Total size: {0}"
+msgstr "Tamaño total: {0}"
+
+#: src/renderer/pages/home/news_feed/news_feed.messages.ts:5:19
+msgid "Try again"
+msgstr "Intentar de nuevo"
+
+#: src/renderer/pages/settings/game_settings/game_settings.messages.ts:15:18
+msgid "Unknown"
+msgstr "Desconocido"
+
+#: src/renderer/pages/settings/game_settings/game_settings.messages.ts:17:22
+msgid "Unvalidated"
+msgstr "Sin validar"
+
+#: src/renderer/app/header/play_button/play_button.messages.ts:3:19
+msgid "Updating"
+msgstr "Actualizando"
+
+#: src/renderer/pages/settings/game_settings/game_settings.messages.ts:14:16
+msgid "Valid"
+msgstr "Válido"
+
+#: src/renderer/pages/settings/game_settings/game_settings.messages.ts:6:20
+msgid "Verifying..."
+msgstr "Verificando..."
+
+#: src/renderer/pages/home/news_feed/news_article/news_article.messages.ts:2:24
+msgid "View on Bluesky"
+msgstr "Ver en Bluesky"
+
+#: src/renderer/pages/replays/replay_browser/replay_browser.messages.ts:3:39
+#. {0} count: number
+msgid "{0, plural, one {# file} other {# files}} filtered."
+msgstr "{0, plural, one {# archivo filtrado} other {# archivos filtrados}}."
+
+#: src/renderer/pages/replays/replay_browser/replay_browser.messages.ts:2:38
+#. {0} count: number
+msgid "{0, plural, one {# file} other {# files}} found."
+msgstr "{0, plural, one {# archivo encontrado} other {# archivos encontrados}}."
+
+#: src/renderer/pages/replays/replay_browser/replay_browser.messages.ts:4:38
+#. {0} count: number
+msgid "{0, plural, one {# file} other {# files}} had errors."
+msgstr ""
+"{0, plural, one {# archivo tenía errores} other {# archivos tenían "
+"errores}}."
```

Comment:
This could just be `"{0, plural, one {# archivo tenía} other {# archivos tenían}} errores."`, as the word "errores" doesn't change

Also, I'm not sure why there's a a empty string at the beginning there, don't really know if necessary given that the original doesn't have it

Reply:
Fixed! The empty strings just indicate that the string is split across multiple lines. It doesn't make a functional difference if it's split across multiple lines or not.